### ISSUE TITLE
 Clear Cache keeps spinnig after Joomla update 5.2.0

### ISSUE THREAD

**Issue by Dnsftl** [templaza/astroid-framework#833]
@sonvnn 

Bug Description
There is a minor bug in Astroid that I wanted to bring to your attention. After updating Joomla to version 5.2.0 and using Astroid 3.1.6 on our development website, the spinner of the "Clear Cache" button continues to spin even after a message indicates that the cache has been successfully cleared. The rest of Astroid appears to work as it should. I have no idea if this issue occurs for others as well. 

Steps to Reproduce
Update Joomla to version 5.2.0.
Click on the "Clear Cache" button in the back end.
Observe that the spinner for "Clear Cache" continues to spin, despite the confirmation message.
Go to the template settings in Astroid and try clearing the cache again to see if the issue occurs there as well.

Expected Behavior
After clicking the "Clear Cache" button, I expected the spinner to stop spinning, as there is a confirmation message.

Screenshots/video
https://github.com/user-attachments/assets/7b5a2807-26f9-479e-a90c-c6dbfe8ba41d

System Information
Joomla Version: 5.2.0
PHP Version: 8.3
Astroid Version: 3.1.6
Device: Laptop
Operating System: Windows 11
Browsers: Chrome, Firefox, Edge

Additional Context
All other functions seem to be working normally, except for the "Clear Cache" button in Astroid. There is no urgency to address this, especially since everything else seems to be functioning properly.

Regards,

Dennis

**Comment by sonvnn** [comment]
@Dnsftl Thank you for report issue. I fixed this issue at https://github.com/templaza/astroid-framework/commit/1303c6d4220a32620334dd350791b170a44535b1

Please help me download v3-branch and retest it.

Best Regards,
Sonny

**Comment by Dnsftl** [comment]
@sonvnn,

It is now working as it should; you can mark it as resolved.

Regards,

Dennis

**Comment by salahjaafar** [comment]
Yes, it works with version 3.1.7 using PHP 8.2, XAMPP server, Linux Mint OS, and Joomla 5.2
